mat_dp_pipeline/sdf/standard_data_format.py

```diff
@@ -215,19 +239,19 @@ def dfs(root: Path) -> StandardDataFormat | None:
             logging.warning(f"No targets found in {root.name}. Ignoring.")
             return
         else:
-            # *Move* metadata from all intensity frames into tech_meta
-            tech_meta_cols = ["Description", "Material Unit", "Production Unit"]
+            # *Move* metadata from all intensity frames into tech_metadata
+            tech_metadata_cols = ["Description", "Material Unit", "Production Unit"]
             all_intensities = list(intensities_yearly.values()) + [intensities]
-            all_meta = [
-                i.loc[:, tech_meta_cols] for i in all_intensities if not i.empty
+            all_metadata = [
+                i.loc[:, tech_metadata_cols] for i in all_intensities if not i.empty
             ]
-            if all_meta:
-                tech_meta = pd.concat(all_meta).groupby(level=(0, 1)).last()
+            if all_metadata:
+                tech_metadata = pd.concat(all_metadata).groupby(level=(0, 1)).last()
             else:
-                tech_meta = pd.DataFrame()
+                tech_metadata = pd.DataFrame()
 
             for inten in filter(lambda df: not df.empty, all_intensities):
```

Comment:
inten -> intensities

Reply:
Already used. It will work, but probably not the best. `df` ?

Reply:
What is intensities vs all_intensities vs inten?

Reply:
Think we might need more specific terms but need to know what each is first

Reply:
`all_intensities = list(intensities_yearly.values()) + [intensities]`. `inten` is an element of `all_intensities` which isn't empty.